### PR TITLE
TinyMCE 5.9 Enterprise release

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-873

Description of Changes:
* TinyMCE 5.9 Enterprise release

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/misc/supported_versions.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Supported versions of {{site.productname}}:
 
 | Version |   Release Date    |  End of Support   |
 |:-------:|:-----------------:|:-----------------:|
+|   5.9   | TBD               |        TBD        |
 |   5.8   | May 19, 2021      |        TBD        |
```
Comment (lnewson) [inline _includes/misc/supported_versions.md:8]: Don't forget to add the end date for 5.8 here.

Comment (tylerkelly13) [reply _includes/misc/supported_versions.md:8]: see https://github.com/tinymce/tinymce-docs/pull/2096

File: _includes/misc/custom-dictionaries-path.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+The `ephox.spelling.custom-dictionaries-path` element is used to define the location of the custom dictionaries. When the setting is not provided, no custom dictionaries are loaded.
+
+Requirements:
+
+- The directory containing the custom dictionaries must be on same server machine as the java service.
+- The directory should not contain subdirectories or non-dictionary files.
+
+{{site.companyname}} recommends storing the custom dictionaries in a similar location to the `application.conf` file. For example, if `application.conf` is in a directory called `/opt/ephox`, the dictionary files could be stored in the subdirectory `/opt/ephox/dictionaries`.
```
Comment (EkimChau) [inline _includes/misc/custom-dictionaries-path.md:8]: /opt/ephox/dictionaries we'll need to keep an eye on this, from memory we have places that use /opt/tiny/dictionaries

**Review by EkimChau — APPROVED** [review]

**Review by JamesToohey — APPROVED** [review]

**Review by lnewson — APPROVED** [review]
Did a skim of a few important pages using launcher and everything seemed okay.

I will note that we're still missing the RTC unsupported APIs, but I've confirmed with the team that, that will be added after release as it's not quite ready yet.

**Pull request merged**